interface/wx/sysopt.h

```diff
@@ -37,6 +37,15 @@
         this option allows changing it without modifying the program code and
         also applies to asserts which may happen before the wxApp object
         creation or after its destruction.
+    @flag{catch-unhandled-exceptions}
```

Comment:
I don't have a strong opinion about this, but it seems a bit strange to have an option which is 1 by default. Maybe we should have `allow-exceptions-escape` or something like that with the opposite meaning instead?

Reply:
I don't think it is that strange. IMO nothing in system options' documentation or the name itself suggests that options are inherently 0 by default. (Also cf. Cocoa's [NSUserDefaults](https://developer.apple.com/documentation/foundation/nsuserdefaults)). In fact, there are two existing system options that are 1 by default: `msw.notebook.themed-background` and `msw.staticbox.optimized-paint`.

Enforcing 0-by-default is bound to lead to more or less awkward naming with made up opposite meanings. E.g. in `allow-exceptions-escape`, 'escape' is not a term that the C++ standard uses with exceptions at all.

Reply:
We could use `disable-catch-unhandled`.

Anyhow, as I said, I don't feel that strongly about this, so if you do, we can leave it like this but I'd still be curious if nobody else really finds this awkward, as I do.